core/common/src/main/java/alluxio/grpc/GrpcChannelBuilder.java

```diff
@@ -70,6 +70,17 @@ public static GrpcChannelBuilder newBuilder(GrpcServerAddress address,
     return new GrpcChannelBuilder(address, conf);
   }
 
+  /**
+   * Sets human readable name for the channel's client.
+   *
+   * @param clientType client type
+   * @return the updated {@link GrpcChannelBuilder} instance
+   */
+  public GrpcChannelBuilder setClientType(String clientType) {
+    mChannelKey.setClientType(clientType);
```

Comment:
if unset it is empty string? Is there a way to know if all channel keys set the client type?

Reply:
We could assert that it's assigned. I saw it as an auxiliary field so didn't enforce it. Do you think we should always supply this field?

Reply:
no, i don't think it has to be required.

Reply:
will leave as it is for now then.